sirepo/package_data/template/warpvnd/egun.py.jinja

```diff
@@ -72,11 +72,11 @@ if can_write_files:
         particles[0][2].append(yvals)
 
     update_status_file(ITERATIONS + 1)
-    with h5py.File('{{ particleFile }}', 'w') as hf:
-        dict_to_h5(
-            {
-                'kept': particles[0],
-                'lost': particles[1]
-            },
-            hf, path='particle'
+    with open("{{ particleFile }}", "wb") as f:
+        pickle.dump(
+            dict(
+                kept=particles[0],
+                lost=particles[1],
+            ),
+            f,
```

Comment:
Is there a way to use something other than pickle? It's known to be insecure

Reply:
Do not use pickle. If you need density, use msgpack or hdf5 otherwise use json.

Reply:
There's also an allow_pickle=True above. Do not use pickle for this.

Reply:
I'll update it to use msgpack. hdf5 doesn't work because the array sizes are all different.

Reply:
Why is the allow_pickle necessary?

Reply:
I don't know why that was added - possibly there was an object in the data. I'll see if it can be taken out.